test.sh

```diff
@@ -93,6 +94,14 @@ assertError() {
   assertEquals 'error msg' "$1" "$( show_output | jq -r .error.message )"
 }
 
+extractUid() {
```

Comment:
Should we maybe just make a `responseJq` method instead of a myriad of `extractSomething`s? I'd be ok with writing `assertEquals "$TEST_EMAIL1" "$( responseJq .email )"`.

Reply:
I don't see much benefit of `responseJq` over inlining `show_output | jq` which is used in some places already. Mostly I didn't want to duplicate the actual path bit in multiple places, which is why I've been creating helpers when the same piece of information is used in multiple tests, but once the api stabilizes we shouldn't need to change them much anyway so it's all a bit of a wash.